sidebarsTHEOads.ts

```diff
@@ -0,0 +1,39 @@
+import type { SidebarsConfig } from '@docusaurus/plugin-content-docs';
+
+const sidebars: SidebarsConfig = {
+  theoads: [
+    'index',
+    {
+      type: 'category',
+      label: 'Getting started',
+      description: 'Set up your first THEOplayer in just a few minutes!',
```

Comment:
This needs changing.
```suggestion
      description: 'Set up your first stream with THEOads',
```